.github/workflows/codespell.yml

```diff
@@ -7,8 +7,7 @@ on:
   pull_request:
     branches: [master]
 
-permissions:
-  contents: read
+permissions: {}
 
```

Comment:
The permission change needs to be documented in the commit message. I see that it disables all. This comes from https://github.com/libfuse/libfuse/issues/792 - what speaks against read permissions?

Reply:
Whilst this check does read from the repository it does so anonymously without using the token so my takeaway was that the token could be granted with no permissions.